Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,21 @@
 function sum(elements) {
+  //it checks if the parameter is an array, if not it returns the message 'not an array'
+  if (!Array.isArray(elements)) return "not an array";
```

Comment:
How did you decide to return a string here? Can you think of any other ways you could signal that there's no good answer to the question being asked of the function?